lib/src/utils/utils.dart

```diff
@@ -1,3 +1,4 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
```

Comment:
If we continue to develop this app, the Util class will become fat: separate it into dialog, url (constant) and timestamp class.

Reply:
@mafreud  dynamic because it have two type return:
- Timestamp if param not null
- FieldValue.serverTimestamp if param null

Reply:
Ok, I got it